fixtures/test_data.json

```diff
@@ -935,13 +926,5 @@
       "content_type": 16
     }
   }
-  {
-    "pk":1,
-    "model": "core.PedigreeRelationshipRole",
```

Comment:
Why was this fixture removed?

Reply:
I wasn't able to find the model core.PedigreeRelationshipRole in ehb_service/apps/core/models/identities.py so this fixture was giving an error when we ran 'python bin/manage.py loaddata  fixtures/test_data.json'. We do, however, have these following models related to relationships: 
- core.PedigreeSubjectRelation
- core.Relationship  
- core.ExternalRecordRelationship -> I think this is closest to the PedigreeRelationshipRole?